src/core/attestation.ts

```diff
@@ -0,0 +1,36 @@
+import { base } from 'viem/chains';
+import { AttestationSchema } from './types';
+import type { Chain } from 'viem';
+
+export const easScanGraphQLAPI = 'https://base.easscan.org/graphql';
+
```

Comment:
Let's split this file in multiple files, where the main one from `core` is the name of the function exported, and anything else goes into utils or store.